src/components/ToolTip.svelte

```diff
@@ -0,0 +1,43 @@
+<script context="module">
+  import HelpStrings from "../data/help";
+
+  export const getHelpMessage = (key) => {
+    if (HelpStrings[key]) {
+      return `<span>${HelpStrings[key].text}`;
+    }
+    return "Specific information not found";
+  };
+
+  export const getHelpLink = (key) => {
+    if (HelpStrings[key]) {
+      return HelpStrings[key].link === undefined
+        ? ""
+        : `${HelpStrings[key].link}`;
+    }
+    return "No link found.";
+  };
+</script>
+
+<script>
+  import tippy from "sveltejs-tippy";
+
+  export let props = {
+    content: "<span class='tooltip'>Tooltip Info</span>",
+    allowHTML: true,
+    placement: "top",
+  };
+</script>
+
+<style>
+  .main {
+    @apply inline-block;
```

Comment:
Might make sense to use a more subtle shade than blue for the help icon.

```suggestion
    @apply inline-block;
    @apply text-gray-500;
```